package.json

```diff
@@ -55,6 +56,7 @@
     "file-loader": "^0.9.0",
     "html-loader": "^0.4.4",
     "html-webpack-plugin": "^2.23.0",
+    "http-server": "^0.10.0",
```

Comment:
Why do you need this?

Reply:
This is used in the command `npm run travis` - we use it to serve the dist directly rather than using `webpack-dev-server` to build and serve the SDK.